### ISSUE TITLE
https://hotosm.org redirect required

### ISSUE THREAD

**Issue by IknowJoseph** [hotosm/hotosm-website#319]
http://www.hotosm.org redirects to https://www.hotosm.org (301)
http://hotosm.org redirects to https://www.hotosm.org (301)
https://hotosm.org fails

**Comment by IknowJoseph** [comment]
An example of a page that features an affected link: https://www.hotosm.org/updates/2016-05-05_hot_annual_meeting_and_election_results

We can edit that post :) but there may be external links we can't update.

**Comment by smit1678** [comment]
@IknowJoseph thanks for this catch. We'll handle this via AWS and point https://hotosm.org --> https://www.hotosm.org. 

cc @dakotabenjamin

**Comment by smit1678** [comment]
Ok, this has been fixed. Here's what I did: 

- set up redirect all request on `hotosm.org` bucket
- put a Cloudfront deployment in front of S3 bucket to pass both http and https traffic to `hotosm.org`
- now all `hotosm.org` traffic is redirected to `www.hotosm.org`. 

We can get more detailed in how we handle all our redirects but for now we're redirecting all traffic. https://docs.aws.amazon.com/AmazonS3/latest/dev/how-to-page-redirect.html
